content/modules/redisbloom/redisbloom-quickstart.md

```diff
@@ -13,9 +13,11 @@ For this quick start tutorial, you need:
     - A Redis Cloud Pro database [with the RedisBloom module](https://redislabs.com/redis-enterprise-cloud/)
 - redis-cli with connectivity to a redis database
 
+You can [set up a free Redis Cloud database]({{< relref "/modules/modules-quickstart.md" >}}) to see the module in action.
+
```

Comment:
Why not add this as a third option to the bulleted list that comes before it?

Reply:
I added it as a paragraph under the first option